app/components/procedure/import_component.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class Procedure::ImportComponent < ApplicationComponent
+  def initialize(procedure:)
+    @procedure = procedure
+  end
+
+  def scope
+    @procedure.routing_enabled? ? 'groupes' : 'instructeurs'
+  end
+
+  def template_url
```

Comment:
```suggestion
  def template_file
```

+ j'ai l'impression que ça peut passer en private